tools/transformer/src/transformations/oneOf.js

```diff
@@ -135,15 +136,30 @@ function handleDuplicates(parentObject, childObject) {
   ]);
   if (duplicates.length > 0) {
     const duplicatesSource = childObject.title || "";
-    const missmatches = duplicates.filter((e) => e.typeMismatch);
+    let missmatches = duplicates.filter((e) => e.typeRefMismatch);
     if (missmatches.length > 0) {
-      throw new Error(
-        `${duplicatesSource} missmatch type detected: ${JSON.stringify(
-          missmatches,
-          undefined,
-          2
-        )}`
-      );
+      missmatches = missmatches.filter(missmatch =>{
+        for(const ignoredProperty of ignoredProperties){
+          if(missmatch.key == ignoredProperty ){
+            console.warn("Type missmatch found when merging base types. Ignoring due to known missmatches", JSON.stringify(
```

Comment:
Once we triage all ignored properties I think we could move logging level to info to avoid excessive output in CI/CD.